### DISCUSSION TITLE
How to capture an image of a rendered element?

### DISCUSSION THREAD

**Discussion by ray007** [MicrosoftEdge/WebView2Feedback#4079] [Q&A]
Is it possible to capture an image of only part of the displayed page?
Maybe somehow with the DevTools protocol?

Reason: drag&drop generates an image for the dragged element, but does not take into account scaling of the view with a css transform (and possibly other transformations). So I'd like to generate an image of the element about to be dragged and use that as drag image.

**Comment by victorhuangwq** [comment]
@srwei do you have an idea about this?
